persistence/sql/persister_session.go

```diff
@@ -76,6 +76,24 @@ func (p *Persister) ListSessionsByIdentity(ctx context.Context, iID uuid.UUID, a
 	return s, nil
 }
 
+func (p *Persister) CountSessionsByIdentity(ctx context.Context, iID uuid.UUID, active *bool) (int64, error) {
```

Comment:
You can avoid this whole function and instead use the `(*pop.Query).Paginator.TotalEntriesSize` form the original https://github.com/ory/kratos/blob/439f015825d704e395d1c23127ba968eda9c961d/persistence/sql/persister_session.go#L44 query similar to: https://github.com/gobuffalo/pop/blob/8919b888d580830e5f04e23b56bc860a07a2f7eb/pagination_test.go#L58-L67
So changing the signature of `ListSessionsByIdentity` to return `(sessions []*session.Session, totalCount int64, err error)`